packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/renderNode.tsx

```diff
@@ -1,3 +1,4 @@
+// Copyright (c) 2020 The Jaeger Authors.
```

Comment:
We don't have a firm policy. Some lawyers told me that the "correct" way to update copyrights is to add a year when the change is made, so one could end up with `Copyright (c) 2017,2019,2020 The Jaeger Authors.` Which seems rather heavy handed and somewhat pointless since one needs to go through Git history to figure out which parts of the code are C in which year. 

A simpler solution is to use a range `2017-2020`.

Simple replacing C with the current year appears to be wrong, but still done in many oss projects (and tooling).

So I would go with the range, if there was a year before.

Reply:
Done, changed to use range from previous date but now is set to Jaeger Authors.